resolver/to_json.py

```diff
@@ -1,11 +1,17 @@
+import json
+
+from sceptre.exceptions import SceptreException
 from sceptre.resolvers import Resolver
 
 
-class CustomResolver(Resolver):
+class ToJsonResolver(Resolver):
     def resolve(self):
         """
         resolve is the method called by Sceptre. It should carry out the work
         intended by this resolver. It should return a string to become the
         final value.
         """
-        return self.argument
+        try:
+            return json.dumps(self.argument)
```

Comment:
Same comment here: it needs to support 1-element lists in order to support nested resolvers.